src/backup/BackupPhraseContainer.tsx

```diff
@@ -12,6 +12,25 @@ import Logger from 'src/utils/Logger'
 
 const PhraseInput = withTextInputPasteAware(TextInput, { top: undefined, right: 12, bottom: 12 })
 
+type TwelveWordTableProps = {
+  words: string
+}
+
+function TwelveWordTable({ words }: TwelveWordTableProps) {
+  return (
+    <>
+      {words.split(' ').map((word, index) => (
```

Comment:
nit: you already split the words to figure out if there are 12 words and this component should be used in the first place. would be more efficient to take a list of words as props, and avoid splitting the same string twice.